app/static/datasets/dataset-cbrain-ids.json

```diff
@@ -0,0 +1,15 @@
+{
+	"1000 Genomes Project":"https://portal.cbrain.mcgill.ca/groups/switch?id=8481",
+	"BigBrain dataset":"https://portal.cbrain.mcgill.ca/groups/switch?id=7686",
```

Comment:
Was only the BigBrain dataset installed? There is not 3DClassifiedVolumes or 3DSurfaces in CBRAIN?

Reply:
well this is the directory in CBRAIN which I should see the installed datasets (projects) https://portal.cbrain.mcgill.ca/search?utf8=%E2%9C%93&search=conp
and I saw just BigBrain not the other ones